index.js

```diff
@@ -91,9 +99,20 @@
       }
 
       previousSpiderLegs = spiderLegs;
+
+      if (clusterElement) {
+        clusterElement.dispatchEvent(
+          new CustomEvent('spiderify', {
+            spiderLegs
+          })
+        )
+      }
+
+      options.onSpiderify(previousSpiderLegs)
```

Comment:
I would suggest, we return the spiderLegs on `spiderfy` and `unspiderfy` method calls. Since they are called from outside the library and i do not understand the need to emit these as events. For example:

```
   spiderifier.onSpiderfy = (args) => {
       ....
   }
   spiderifier.spiderfy([-74.50, 40], features);
```

```
   const spiderLegs =  spiderifier.spiderfy([-74.50, 40], features);
   onSpiderfy(spiderLegs);
   clusterElement.dispatchEvent(
       new CustomEvent('spiderify', {
          spiderLegs
       })
   );
```

The clusterElement seems like a implementation specific one and not generic for the library.

Reply:
Totally up to you. Yes the clusterElement implementation was because, in the particular project, we wanted to manipulate how the original cluster marker looked when it was spiderified vs not. But I can remove for the library. 

I'm not sure I quite understand what you're asking in suggesting returning the spiderLegs. But I attempted to adjust.

Reply:
Return the spiderLegs in the spiderfy and unspiderfy function in the library. That way when you call spiderfy or unspiderfy, you can get the legs and do what you want with it